README.md

```diff
@@ -148,6 +149,15 @@ Feel free to check the [issues page](../../issues/).
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
+## © Acknowledge <a name="acknowledge"></a>
+
+Thanks to Muhamad Fandi, he is the one who create the amazing background image, you can [click here](https://www.vecteezy.com/members/fandi-creations) to discover more of his work.
+
+Also, you can find more background like this in [vecteezy](https://www.vecteezy.com/free-vector/pixel-game-background).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
- [ ] OPTIONAL
MIT license link in your ReadMe is broken, kindly endeavor to fix it, and remove the guide note below it, from _line 185 - 188_.

Reply:
Thanks @rloterh, I'll do it right away.
Have a great day.